spec/person_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative '../person'
```

Comment:
- [ ] Kindly add tests for all classes you implemented in the Decorator design pattern ( Nameable, BaseDecorater, CapitalizeDecorator, TrimmerDecorator ) as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md#:~:text=all%20classes%20that%20you%20created%20while%20implementing%20the%20Decorator%20design%20pattern).